lib/cocoapods/installer/analyzer.rb

```diff
@@ -421,23 +421,21 @@ def generate_targets(resolver_specs_by_target, target_inspections)
       # @return [AggregateTarget]
       #
       def generate_target(target_definition, target_inspections, pod_targets, resolver_specs_by_target)
+        target_requires_64_bit = requires_64_bit_archs?(target_definition.platform)
         if installation_options.integrate_targets?
           target_inspection = target_inspections[target_definition]
           raise "missing inspection: #{target_definition.name}" unless target_inspection
           user_project = target_inspection.project
           client_root = user_project.path.dirname.realpath
           user_target_uuids = target_inspection.project_target_uuids
           user_build_configurations = target_inspection.build_configurations
-          archs = target_inspection.archs
+          archs = target_requires_64_bit ? ['$(ARCHS_STANDARD_64_BIT)'] : target_inspection.archs
```

Comment:
I guess it does not matter those are all the pod targets... but worth just brining it up

Reply:
right now we read the setting from the user project (if one is explicitly set) - this will override that, yes, but that's sort of what we want because by default the archs setting doesn't have an explicit value

Reply:
I just realized that this is probably what is causing the integration spec changes, not the other section. this section never had the check for `:osx`

Seems like that's a bug that is now fixed?

Reply:
aaah yes